### DISCUSSION TITLE
Inquiry about "Authenticator" for GoogleChrome

### DISCUSSION THREAD

**Discussion by Kondo-CSE** [Authenticator-Extension/Authenticator#791] [Q&A] [answered]
Hi,
I am contacting you to enquire about "Authenticator" for GoogleChrome.

The overview of "Chrome Web Store" site says "Authenticator generates two-factor authentication (2FA) codes".
Is it correct that it complies with RFC6238 ?

I contacted you to see if "Authenticator" can be used as Salesforce multi-factor authentication.

**Comment by Sneezry** [comment, marked as answer]
Hi Koji Kondo,

"Authenticator" complies with RFC6238. See https://authenticator.cc/docs/en/overview

We haven't tested for Salesforce account, but I see someone suggests "Authenticator" on Salesforce website: https://trailhead.salesforce.com/en/trailblazer-community/feed/0D54S00000B3vi3SAB

**Reply by Kondo-CSE** [reply]
Hi Zhe Li,

Thank you for your reply.
I have understood the content.
